packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -117,15 +145,248 @@
       }
     }
   }
-  const onChange = Utils.sequential(async (e, key) => {
+
+  $: customStepLayouts($memoBlock, schemaProperties)
```

Comment:
This is basically what I'm suggesting to use in `RowSelector` as well - a nice function with explicit params to react to rather than a monster reactive statement.